TrainingContainer.Dockerfile

```diff
@@ -1,11 +1,30 @@
 FROM python:3.9-slim
```

Comment:
This isn't your fault but @karkir0003 do you think we should instead use the full version of python instead of the slim version? I feel like it'll have more functions

Reply:
But wouldn't the docker image then be a bit heavier? @farisdurrani. I think that the slim version anyway comes with pip

Reply:
That is true. I guess if it works, it's fine